Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:buster
+
+RUN apt-get update
+RUN apt-get -y install curl git rsync \
+    # Node version 14
+    && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+    && apt-get install nodejs
+
+
+RUN git clone https://github.com/codetheweb/MTU-Transfer-Course-Gatherer.git /app/
+WORKDIR /app/
+
+# Install Python dependencies
+
+RUN pip3 install --upgrade pip -r requirements.txt
+RUN python3 ./manage.py makemigrations
```

Comment:
For clarification's sake, do you mean a Docker volume?

Reply:
Yeah, sorry. Just so we can have a persistent DB between image changes.

Reply:
I think for the sake of expansion I'd like to move this to another issue. Ideally, I'd like to use a real DBMS instead of SQLITE, and have that hosted in another container. (Maybe the same container for dev?)

Reply:
👍 splitting stuff into Dockerfile.dev and Dockerfile works for me.

Maybe for the prod Dockerfile we could [omit the seed data](https://github.com/Kiro47/MTU-Transfer-Course-Gatherer/pull/33#discussion_r419166234).

So Dockerfile.dev would be batteries-included quickstart, Dockerfile would be recommended for prod envs.

Reply:
Created Issue #35